truss/truss_config.py

```diff
@@ -501,8 +501,7 @@ class TrussConfig:
     def canonical_python_version(self) -> str:
         return {
             "py39": "3.9",
-            "py38": "3.8",
-            "py37": "3.7",
+            "py311": "3.11",
```

Comment:
do we need py310 in here to? I don't really see where this is being used actually.

Reply:
Hmm, it's included in `SUPPORTED_PYTHON_VERSIONS` though.